src/SmartComponents/DriftPage/DriftTable/ComparisonHeader/ComparisonHeader.js

```diff
@@ -52,28 +63,23 @@ class ComparisonHeader extends Component {
         }
 
         return sortIcon;
-    }
-
-    async toggleSort(sortType, sort) {
-        const { setHistory, toggleFactSort, toggleStateSort } = this.props;
+    };
 
+    const toggleSort = async (sortType, sort) => {
         if (sortType === 'fact') {
             await toggleFactSort(sort);
         } else {
             await toggleStateSort(sort);
         }
 
         setHistory();
-    }
+    };
 
-    renderLoadingSystems() {
+    const renderLoadingSystems = () => {
```

Comment:
This looks also like it should be a component on its own.

Generally, if it returns a component/JSX it should be a component.